### PR TITLE
WHO=18 Energy Manager support

### PR DESCRIPTION
**Pull request by aconte80**
Contribution for supporting WHO=18 own messages

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+
+## [0.4.0] - 2021-02-08
```
Comment (mvalla) [inline CHANGELOG.md:4]:
```suggestion
## [0.4.0-SNAPSHOT] - 2021-02-??
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+### Added
+- Initial support for Energy Manager (WHO=18)
```
Comment (mvalla) [inline CHANGELOG.md:7]:
```suggestion
- Initial support for Energy Management (WHO=18)
```

File: pom.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 	<groupId>com.github.openwebnet4j</groupId>
 	<artifactId>openwebnet4j</artifactId>
-	<version>0.3.4</version>
+	<version>0.4.0</version>
```
Comment (mvalla) [inline pom.xml:8]:
```suggestion
	<version>0.4.0-SNAPSHOT</version>
```

File: src/test/java/org/openwebnet4j/test/MessageTest.java [hunk 4]
```diff
@@ -237,4 +242,43 @@ public void testGatewayMgmt() {
...
+            assertEquals(EnergyManagement.DIM.ACTIVE_POWER, energyMsg.getDim());
+            assertNotNull(energyMsg.getDimValues());            
+        } catch (FrameException e) {
+            System.out.println(e.getMessage());
+
+            Assertions.fail();
+        }
+    }
+
+    // @Test
```
Comment (mvalla) [inline src/test/java/org/openwebnet4j/test/MessageTest.java:264]: i suggest to remove commented code, MessageTest is intended for unit tests, this looks more as an integration test (it involves communication with real gw)

**Review by mvalla — COMMENTED** [review]
Thanks @aconte80 for the contribution! 
Just minor comments : see my review. 
We target version 0.4.0 but since I will add few more commits before release, we should leave here 0.4.0-SNAPSHOT

File: src/test/java/org/openwebnet4j/test/MessageTest.java (hunk 4 above)
Comment (aconte80) [reply src/test/java/org/openwebnet4j/test/MessageTest.java]: removed

**Comment by aconte80** [comment]
Fixes #8

**Pull request merged**